backend/antibiootit/src/main/java/fi/tuni/koodimankelit/antibiootit/services/DataHandlerImpl.java

```diff
@@ -30,5 +30,10 @@ public Diagnose getDiagnoseById(String id) {
     public List<DiagnoseInfo> getAllDiagnoseInfos() {
         return repository.getAllDiagnoseInfos();
     }
+
+    @Override
+    public DiagnoseInfo getDiagnosisInfoById(String id) {
+        return repository.getDiagnosisInfoById(id);
```

Comment:
Tässä pitäis muuttaa repository -> diagnoseRepository.

Reply:
Muutettu!